client/src/components/CategoryIndex.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from 'react'
+import CategoryTile from './CategoryTile'
+const CategoryIndex = props => {
```

Comment:
We can add some line breaks between these lines. A good rule to follow is to first group together all external `import` statements (like react), add a new line break, group together all internal `import` statements (like components), add a new line break, then define the component itself